.github/labels.md

```diff
@@ -0,0 +1,249 @@
+sense to check if the label is appropriate or not for the specific Issue.
+
+Currently there are three categories:
+
+
+#### Label category `category`
+
+Labels within this category describe the intent of the Issue.  Usually there
+should be exact one intent for creating an Issue and it should not change during
+the lifetime of an Issue.
+
+#### Label category `component`
+
+The component lists the parts of the project this Issue affects or is affected
+by.  This should help developers to identify if they have required knowledge or
+interests to handle the issue.  There should be at least one component listed
+for an Issue.
+
+#### Label category `status`
+
+With the status category developers, managers or issue reporters can check if
+an issue needs further interaction and who should carry out this interaction.
+Issue have exactly one status.  Status may change over the lifetime of the
+Issue.
+
+
+## Labels
+
+This section lists when an label should be applied and what additional actions
+may be taken.
+
+
+#### Label `category:bug`
+
+The Issue describes or solves a perceived malfunction within the game.
+
+#### Label `category:feature`
+
+The Pull Request contains a new game feature.  Features are parts of the game
+which a player can interact with e.g.: A new species, a new gameplay mechanic
+or a new command line switch for the server that changes gameplay behavior.
+
+#### Label `category:idea`
+
+??? No idea, overlaps with `category:feature` or `state:work in progress` IMO.
+
+#### Label `category:question`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:refactoring`
+
+The Pull Request contains a new or improved implementation of an existing
+feature or any other code restructuring that doesn't modify the current game
+mechanics or representation.
+
+#### Label `category:request`
+
+??? Requests don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+The Pull Request contains insignificant code changes that don't involve
+features, refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue deals with the python AI decision making code or affects it.
+
+#### Label `component:art music text assets`
+
+The Issue deals with creating game assets or verification if the assets match
+the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue deals with CMake, Visual Studio, Xcode or the general build process.
+
+#### Label `component:common`
+
+??? No idea.
+
+#### Label `component:deployment`
+
+The Issue deals with the deployment process or the software used create the
+installer.
+
+#### Label `component:documentation`
+
+The Issue deals with Doxygen or primarily contains documentation.
+
+#### Label `component:game mechanic`
+
+The Pull Request deals with the currently used or planned game mechanics.  The
+content of the Pull Request may require extensive gameplay balance testing.
```

Comment:
Same as above: why only referring to "Pull Request" instead of "issue" (which should cover both)?